db/migrate_cards/data/cards/homepage_metric_company_item

```diff
@@ -0,0 +1,19 @@
+<div class="yinyang-row">

```

Comment:
just a comment: we're trying to move away from yin-yang row.  We don't have to move quickly, but whenever you work in this area, try to make sure you're using it less and less.